parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -398,7 +398,11 @@ ensureArtifactStreaming() {
   systemctl link /opt/overlaybd/snapshotter/overlaybd-snapshotter.service
   systemctlEnableAndStart overlaybd-tcmu.service 30
   systemctlEnableAndStart overlaybd-snapshotter.service 30
-  systemctlEnableAndStart acr-nodemon 30
+  
+}
+
+ensureAcrNodeMon() {
+    systemctlEnableAndStart acr-nodemon 30
```

Comment:
so we just needed a delay between starting overlaybd services and this one specifically?